File: cognipy/CogniPyLib/CNL/Populator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Ontorion.CNL.DL
{
    public interface Populator
    {
        IEnumerable<KeyValuePair<string, string>> Populate(string sentenceBeginning, string str, List<string> forms, int max);
    }
}

File: cognipy/cognipylib/cognipy/cnl/dl/Populator.java
package cognipy.cnl.dl;

import cognipy.*;
import cognipy.cnl.*;
import java.util.*;

public interface Populator
{
	java.lang.Iterable<Map.Entry<String, String>> Populate(String sentenceBeginning, String str, ArrayList<String> forms, int max);
}